### PR TITLE
clean up Gordon's prompt

### PR DESCRIPTION
**Pull request by averyschreiner**

### CODE REVIEW

**Comment by averyschreiner** [comment]
excludes lists that are empty/null from the prompt. no longer sorts previous recipes by meal type, only their feedback

File: Services/JobService.cs [hunk 1]
```diff
@@ -202,63 +210,26 @@ private string GetDietaryConsiderationsText(string familyId)
...
-        var enjoyedLunch = new List<string>();
-        var enjoyedDinner = new List<string>();
-        var indifferentBreakfast = new List<string>();
-        var indifferentLunch = new List<string>();
-        var indifferentDinner = new List<string>();
-        var notEnjoyedBreakfast = new List<string>();
-        var notEnjoyedLunch = new List<string>();
-        var notEnjoyedDinner = new List<string>();
+        var enjoyed = new List<string>();
+        var notEnjoyed = new List<string>();
```
Comment (Pchristy03) [inline Services/JobService.cs:214]: How are we determining what meal is what on the overview page with these changes?

**Review by Pchristy03 — APPROVED** [review]
Love the clean up! Just left one comment.

File: Services/JobService.cs (hunk 1 above)
Comment (Pchristy03) [reply Services/JobService.cs:214]: Ohh are they just not sorted on the frontend now?

Comment (averyschreiner) [reply Services/JobService.cs:214]: the mealType is still there for the recipes but we just don't use them when building the prompt. all we are concerned about for each recipe is if they enjoyed them or not. for the overview page we still can access the mealtype for sorting purposes

Comment (Pchristy03) [reply Services/JobService.cs:214]: Ahh gotcha, nice

**Pull request merged**